mkosi/distributions/debian.py

```diff
@@ -42,6 +42,7 @@ def tools_tree_packages(cls) -> list[str]:
             "ca-certificates",
             "coreutils",
             "cpio",
+            "curl"
```

Comment:
Missing comma

Reply:
The vagaries of implicit string concatenation.